src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = React.useState<Person[]>([]);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [error, setError] = React.useState(false);
```

Comment:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
  const [isLoading, setIsLoading] = useState(false);
  const [error, setError] = useState(false);
```
диструктуризуй в імпорті з react